app/javascript/packs/LoginPage.js

```diff
@@ -12,10 +12,10 @@ import {safeFromJsonString} from 'utilities/json-utils'
 document.addEventListener('DOMContentLoaded', () => {
   const oldLoginWrapper = document.getElementById('old-login-page-wrapper')
   oldLoginWrapper.removeChild(document.getElementById('old-login-page'))
-  const loginPageContainer = document.getElementById('login-page-container')
+  const PFLoginPageContainer = document.getElementById('pf-login-page-container')
```

Comment:
@didierofrivia This is the container for the `Patternfly LoginPage` Component. Inside the component goes the login, request password or signup form.
I renamed it to PFLoginPageContainer to make ir more clear

Reply:
Why the change of name anyway?